plugins/processors/cumulative_sum/cumulative_sum.go

```diff
@@ -0,0 +1,146 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package cumulative_sum
+
+import (
+	_ "embed"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type CumulativeSum struct {
+	Log               telegraf.Logger
+	Fields            []string        `toml:"fields"`
+	DropOriginalField bool            `toml:"drop_original_field"`
+	CleanUpInterval   config.Duration `toml:"clean_up_interval"`
+
+	fieldMap    map[string]bool
```

Comment:
Please use a [`filter`](https://github.com/influxdata/telegraf/blob/ffb4a8cedb22cb9d69813a486517fb93d3452179/filter/filter.go#L30) for this to allow wildcards when specifying fields.